requirements_dev.txt

```diff
@@ -0,0 +1,2 @@
+pytest
```

Comment:
We typically use `test_requirements.txt`, `requirements_dev.txt` is a `bird`-ism.

Reply:
Should I pin the `pytest` version?

Reply:
I generally pin all versions.